src/data-interfaces/HposInterface.js

```diff
@@ -134,6 +139,11 @@ const HposInterface = {
     updateVersion: async () => {
       const result = await hposCall({ method: 'post', path: 'upgrade' })()
       return presentHposStatus(result)
+    },
+
+    hostedHapps: async () => {
```

Comment:
Let's add this to the hpos mocks, so we can run/test this interface out manually with mocks as well as live.